pkg/experiment/metastore/compaction/scheduler/schedule.go

```diff
@@ -42,6 +42,7 @@ func (p *schedule) AssignJob() (*raft_log.AssignedCompactionJob, error) {
 		State: state,
 		Plan:  plan,
 	}
+	p.evicted++
```

Comment:
Not too sure if this should be better called assigned instead (I was a bit confused about it):

```suggestion
	p.assigned++
```

Reply:
Actually, I think this is a bug of sorts. For some reason, I counted assigned jobs toward job queue decrease (thus incrementing the evicted counter), which is not necessarily correct – the job might be abandoned and remain in the queue.

I made the limits more conservative and deleted the line.